site/specs/numbers.json

```diff
@@ -1696,8 +1700,9 @@
         "tags": [
           "/accounts"
         ],
-        "description": "<p>Delete a bulk port-in order with child port-ins.  Deleting a bulk port-in is allowed\nfor 'DRAFT' state only.  Deleting a bulk port-in will delete all DRAFT child port-ins\nassociated with the bulk port-in.  When the bulk port-in is deleted, any child port-in\norders that are not in a draft status are dissociated from the bulk port-in, but not\ndeleted.</p>\n",
-        "operationId": "DELETE /accounts/{accountId}/bulkPortins/{orderid}",
+        "description": "<p>Delete a bulk port-in order with child port-ins.  Deleting a bulk port-in is allowed\nfor 'DRAFT' state only.  Deleting a bulk port-in will delete all DRAFT child port-ins\nassociated with the bulk port-in.  When the bulk port-in is deleted, any child port-in\norders that are not in a draft status are dissociated from the bulk port-in, but not\ndeleted. <a href=\"https://dev.bandwidth.com/docs/numbers/guides/bulkportins\">Here is link to Guides and Tutorials</a></p>\n",
```

Comment:
@hamermike just want to confirm , is html the right syntax to use here?  Not markdown?

Reply:
We can remove all the references to almost all the html tags (<p></p> for instance isn't needed) but as you mentioned in another comment we should update to relative links. I'll toss the voice reference here as well. https://github.com/Bandwidth/api-docs/blob/main/site/specs/voice.json#L3343

Reply:
We should use HTML tags for links instead of markdown syntax right? That's what the voice one has that you linked.

Reply:
Yep, that's correct but most other html tags aren't used paragraph tags for instance. I should have made that more clear.